extensions/amp-story/1.0/test/test-amp-story-request-service.js

```diff
@@ -79,7 +98,7 @@ describes.fakeWin('amp-story-request-service', {amp: true}, (env) => {
       })
       .once();
 
-    await requestService.loadShareConfig();
+    await requestService.loadConfig(shareElement);
```

Comment:
Can you fix this test and call `loadConfig` twice? I expect to fail with your new implementation. It's incorrectly passing right now.

Reply:
The test doesn't really make sense anymore since I migrated the once call to amp-story-share - I will refactor this test to call the loadconfig in amp-story-social share twice.

Reply:
sure, I'll refactor it back to add the once call.